### PR TITLE
NETOBSERV-377 Filtering in topology triggers exact match

### PR DESCRIPTION
**Pull request by jotak**

### CODE REVIEW

File: web/src/model/topology.ts [hunk 1]
```diff
@@ -116,17 +116,17 @@ export const getFilterDefValue = (d: ElementData, t: TFunction) => {
...
-    value = d.namespace ? d.namespace! : d.name!;
+    value = `"${d.namespace ? d.namespace! : d.name!}"`;
   } else if (d.type && d.name) {
     if (['Service', 'Pod'].includes(d.type)) {
       def = findFilter(t, 'name')!;
     } else {
       def = findFilter(t, 'owner_name')!;
     }
-    value = d.name;
+    value = `"${d.name}"`;
```
Comment (jotak) [inline web/src/model/topology.ts:129]: @jpinsonneau I didn't find how to trigger this filter (when I'm filtering on a pod or service, it uses the "Resource" filter anyway, not that one) , but anyway, it seems legit to have the same exact match being done here too, I guess.

Comment (jpinsonneau) [reply web/src/model/topology.ts:129]: On owner scope you will get into this case :wink:

**Review by jpinsonneau — APPROVED** [review]
LGTM, tested and works fine. Thanks for the update :+1:

**Comment by jotak** [comment]
Thanks Julien
/approve

**Pull request merged**